app/config.py

```diff
@@ -9,4 +9,8 @@
 SQLALCHEMY_DATABASE_URI = (
     f"postgresql://{_creds}@{ADMIN_POSTGRES_HOST}:5432/{ADMIN_POSTGRES_DATABASE}"
 )
+
+# This used by db-client and alembic script for migrations
+os.environ["DATABASE_URL"] = SQLALCHEMY_DATABASE_URI
```

Comment:
Should we be setting this in the script, or in the environment we are running the admin service from?

Reply:
I think the idea is to build all the necessary config from the `ADMIN_...` vars - unfortunately the only way of communicating this down to the `db-client` library seems to be setting the env var `DATABASE_URL` ... the `db-client` has a function `get_url()` which returns this value.

Reply:
So you can see from line 10 that the environment specifies the ADMIN vars and the code worries about formatting it up for the rest of the code. This make sense @katybaulch

Reply:
Ah understood, thanks for clarifying